src/gate/parserGnet/parserGnet.h

```diff
@@ -0,0 +1,34 @@
+/**
+ * \brief Implements translation from Liberty format file to GNet.
+ * \author <a href="mailto:anushakov@edu.hse.ru">Aleksander Ushakov</a>
+ */
+
+#pragma once
+
+#include "gate/model/gnet.h"
+
+#include <kernel/yosys.h>
+
+#include <map>
+#include <memory>
+#include <vector>
+
+struct NetData {
+    std::vector<std::unique_ptr<eda::gate::model::GNet>> combNets;
+    std::vector<std::unique_ptr<eda::gate::model::GNet>> memNets;
+};
+
+void translateModuleToGNet(
+    const Yosys::RTLIL::Module *m,
+    NetData &net);
+
+void translateDesignToGNet(
+    const Yosys::RTLIL::Design &des,
+    NetData &vec);
+
+void translateLibertyToDesign(
+    const char* namefile,
+    NetData &vec);
+
+std::vector<uint64_t> truthTab(
+    const eda::gate::model::GNet *net);
```

Comment:
An API is iconsistent: pass inputs either by value or by reference, but do not mix them so some functions are taking pointers and others are taking references.

Moreover, add short descriptions in comments, especially for value-returning functions.